README-development.md

```diff
@@ -1,3 +1,39 @@
+# Operating System
```

Comment:
I run basically all of this from a container, and so does the pipeline (except for Imagefactory for bad reasons).  I think we should recommend that as well; it's mentioned below for the rpm-ostree bits.  In fact we could just recommend pulling `quay.io/cgwalters/coreos-assembler` and running things in there right?

Reply:
Switched ``coreos-assembler`` to use ``podman``